### PR TITLE
dashboard: Add Maintainers List

### PR DESCRIPTION
**Pull request by afinn12**
## Needed 
- [x] Merge #20 before this
  - For instance, the changes to the basePath in deploy.yml/next.config.js is done in that PR. We assume these changes. 

## Description
- Added Maintainer information to the rowExpansionTemplate
  - If none exist, a message appears
  - If multiple exist (across groups as well), all names are listed
  - Clicking on the name takes you to their Github
  - On hover, an overlay panel appear with more information
    - Including: Group, Email, Github, and Slack
      - Github and slack are clickable links 

## Testing
- [x] Tested in dev
- [x] Tested in prod

### Example of Maintainer's List
image

### Example of Overlay Panel
image

### CODE REVIEW

**Comment by sprt** [comment]
To update UI as such:
```
Maintainers:
Rust: NameA, NameB
S390x: NameC, NameD NameA
```

**Comment by afinn12** [comment]
## Description
- Maintainers are listed by group with a drop-down panel displaying their contact information. Github/slack are links. 
image

- Name's won't split, but will still wrap. 
image

File: pages/index.js [hunk 1]
```diff
@@ -117,9 +130,82 @@ export default function Home() {
...
+              ))}
+            </div>
+          ) : (
+            <div>No Maintainer Information Available</div>
+          )}
+        </div>
       </div>
     );
-  };
+  };
```
Comment (a1icja) [inline pages/index.js:208]: Should probably revert this

File: pages/index.js [hunk 2]
```diff
@@ -117,9 +130,82 @@ export default function Home() {
             <div>No Nightly Runs associated with this job</div>
           )}
         </div>
+        <div className="mt-4 p-2 bg-gray-300 w-full">
```
Comment (a1icja) [inline pages/index.js:133]: This is a really long section. Any way to break this up?

File: maintainers.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+## slack: member id, display name, or link to profile?
+
+mappings:
+  # @kata-containers/arch-s390x: BbolroC
+  - regex: ".*s390x.*"
+    owners:
+      - fullname: "Hyounggyu Choi"
+        email: "Hyounggyu.Choi@ibm.com"
+        slackurl: "https://katacontainers.slack.com/team/U0369UMM69M"
+        # slackid: "U0369UMM69M"
```
Comment (sprt) [inline maintainers.yml:20]: Let's remove these comments.

File: maintainers.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        slack: "Hyounggyu Choi"
+        github: "BbolroC"
+        group: "arch-s390x"
```
Comment (sprt) [inline maintainers.yml:23]: The `group` key should be at the same level as `regex`, not inside `owners`.

**Review by sprt — CHANGES_REQUESTED** [review]

File: maintainers.yml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      - fullname: "Fupan Li"
+        email: "lifupan@gmail.com"
+        slackurl: "https://katacontainers.slack.com/team/U059Y60DZPW"
+        # slackid: "U059Y60DZPW"
+        slack: "fupan li"
+        github: "lifupan"
+        group: "rust"
+
+# @kata-containers/amd: AdithyaKrishnan, ryansavino
+  - regex: ".*qemu-(sev|snp).*"
```
Comment (AdithyaKrishnan) [inline maintainers.yml:69]: Hey @afinn12 ! Thanks a ton for your contributions.
In one of our CoCo CI meetings, we decided to take ownership of the ovmf, ovmf-sev and snp-experimental workflows as well in addition to our SEV and SEV-SNP workflows
Could you please include the additional ones too under our names?
I guess ```.*(qemu-(sev(-snp)?|snp(-experimental)?)|ovmf(-sev)?).*``` should work

Comment (afinn12) [reply maintainers.yml:69]: Yes, will the group name still just be "amd" for all of this?

Comment (AdithyaKrishnan) [reply maintainers.yml:69]: Yes, preferably "AMD"

**Review by AdithyaKrishnan — APPROVED** [review]

**Review by stevenhorsman — APPROVED** [review]
I think this looks fine as an initial version, that we can interatively improve. Thanks!

**Review by sprt — APPROVED** [review]

**Pull request merged**